AppCenterIdentity/AppCenterIdentity/MSIdentity.m

```diff
@@ -211,12 +218,20 @@ - (BOOL)loadConfigurationFromCache {
   return NO;
 }
 
-- (void)downloadConfigurationWithETag:(nullable NSString *)eTag {
+- (MSIdentityConfigIngestion *)ingestion {
+  if (_ingestion == nil) {
```

Comment:
I am being nit-picking, but we use `!x` instead of `x == nil` syntax?

Reply:
I actually prefer this way also, why are we using the ivar instead of the property?